src/main/resources/portal.properties.EXAMPLE

```diff
@@ -1,6 +1,15 @@
 # app name
 app.name=cbioportal
 
+# feature toggles
+show_all_cancer_studies=false
+show_twitter=false
+show_dev_channels=false
+show_study_download_list=false
+show_reference_genome_genes=false
+is_private_portal=false
+is_localdb_server=false
```

Comment:
```suggestion
# feature toggles
# show_all_cancer_studies=false
# show_twitter=false
# show_dev_channels=false
# show_study_download_list=false
# show_reference_genome_genes=false
# is_private_portal=false
# is_localdb_server=false
```
ideally it would work like this with all these newly introduced properties commented out. Imagine that if someone upgrades their version of cBioPortal we don't want to bother them to add all these new properties to their existing portal.properties file

Reply:
I don't think I can comment them out since the frontend relies upon them for configuration. But since it's only two feature toggles now, it might not be such a problem.